signature-aggregator/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM debian:11-slim
+COPY signature-aggregator /usr/bin/signature-aggregator
+EXPOSE 8080
+EXPOSE 8081
```

Comment:
Why are we exposing this metrics port here but not in the  relayer?

Reply:
Great question. I did it for the aggregator because it felt like the right thing to do, since there'd be no way to consume metrics without it, but I left things the way they were for the relayer in order to limit the scope of changes.

There's a similar question to be asked about why the relayer sets a `USER`. @cam-schultz [once advised me](https://github.com/ava-labs/avalanche-bridge-v2/issues/135#issue-2228504345) to avoid setting a `USER`.

Reply:
I think we should be consistent and remove it from here since it looks like relayer handles without exposing it directly.

Reply:
after doing a little more research, it looks like `EXPOSE` is purely documentation, with a bit of automation convenience that takes advantage of that documentation.

when you run an image with docker run, there's a `-P` option that will automatically "publish" all of the ports that the image `EXPOSE`s, but you can manually choose to publish that port, or not, or publish different ports, and the image can also listen on other ports (eg if a config file said to use a different port), and the runtime can still publish ports that were never even `EXPOSE`d.

so, it seems completely unnecessary, but good documentation, and potentially a helpful convenience to some users. my vote would be to keep it, but i don't feel that strongly about it.

Reply:
Yeah it all makes sense. What this is documenting is default ports that are probably going to be overwritten by the config that you pass in. But I don't feel strongly either actually so can drop